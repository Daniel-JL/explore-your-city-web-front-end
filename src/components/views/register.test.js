import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import userEvent from '@testing-library/user-event';
import {
  MemoryRouter,
} from 'react-router-dom';
import { createMemoryHistory } from 'history';
import Register from './register';

describe('Register page', () => {
  it('should render correctly', () => {
    const wrapper = renderer.create(
      <MemoryRouter>
        <Register />
      </MemoryRouter>,
    ).toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});
