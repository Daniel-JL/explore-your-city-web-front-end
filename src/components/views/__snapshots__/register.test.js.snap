// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register page should render correctly 1`] = `
<div
  className="sc-pFZIQ cPKlFN"
>
  <h1
    className="sc-crrsfI imaEaq"
  >
    Register
  </h1>
  <div
    className="sc-jrAGrp hUEthe"
  >
    <div
      className="sc-kEjbxe sc-iqHYGH whvJx dgHtcJ"
    >
      <input
        className="sc-bdfBwQ fdZBNX"
        id="first_name"
        name="first_name"
        placeholder="First Name"
        type="first_name"
      />
      <input
        className="sc-bdfBwQ fdZBNX"
        id="last_name"
        name="last_name"
        placeholder="Last Name"
        type="last_name"
      />
    </div>
    <div
      className="sc-kEjbxe whvJx"
    >
      <input
        className="sc-bdfBwQ fdZBNX"
        id="e-mail"
        name="e-mail"
        placeholder="Enter E-Mail"
        type="e-mail"
      />
      <input
        className="sc-bdfBwQ fdZBNX"
        id="password"
        name="password"
        placeholder="Enter Password"
        type="password"
      />
      <input
        className="sc-bdfBwQ fdZBNX"
        id="confirm_password"
        name="confirm_password"
        placeholder="Confirm Password"
        type="confirm_password"
      />
    </div>
  </div>
  <a
    aria-current={null}
    href="/contribute"
    onClick={[Function]}
  >
    <button
      className="sc-gsTCUz sc-eCssSg hbelWT ciBFzU"
    >
      Submit
    </button>
  </a>
</div>
`;
